src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -571,7 +594,13 @@ private void processTile(
 
     // TODO: ZCT
     // int[] zct = reader.getZCTCoords(plane);
-    N5Writer n5 = new N5FSWriter(outputPath.resolve(pyramidName).toString());
+    final N5Writer n5;
```

Comment:
Maybe resolve the path once outside conditionals and construct with switch if we use a file type string above?